### PR TITLE
Add readiness probe to router

### PR DESCRIPTION
**Pull request by grs**
Add liveness and readiness probes to service controller Add readiness probe to config sync (ready once initial download of any certs is complete)

### CODE REVIEW

**Review by ajssmith — APPROVED** [review]
lgtm

built images and observed readiness checks succeeding

File: cmd/service-controller/controller.go [hunk 1]
```diff
@@ -527,6 +527,7 @@ func (c *Controller) initialiseServiceBindingsMap() (map[string][]int, error) {
 	c.ports.inuse(int(types.InterRouterListenerPort))
 	c.ports.inuse(int(types.ConsoleDefaultServicePort))
 	c.ports.inuse(9090) // currently hardcoded in config
+	c.ports.inuse(9191) // currently hardcoded in config
```
Comment (fgiorgetti) [inline cmd/service-controller/controller.go:530]: Should 8182 be added here as well?

File: pkg/kube/containers.go [hunk 2]
```diff
@@ -35,6 +35,37 @@ func ContainerForController(ds types.DeploymentSpec) corev1.Container {
 		ImagePullPolicy: GetPullPolicy(ds.Image.PullPolicy),
 		Name:            types.ControllerContainerName,
 		Env:             ds.EnvVar,
+		Ports:           ds.Ports,
+		StartupProbe: &corev1.Probe{
```
Comment (fgiorgetti) [inline pkg/kube/containers.go:39]: Maybe, in an separate PR/Issue, this could also be done as part of a `skupper update` to existing deployments.

**Review by fgiorgetti — APPROVED** [review]
A few comments only.
But it works OK and looks good.

**Review by nluaces — APPROVED** [review]

File: pkg/kube/containers.go (hunk 2 above)
Comment (grs) [reply pkg/kube/containers.go]: Good point, updated now

**Pull request merged**